src/components/cardbuilder/cardImage.ts

```diff
@@ -0,0 +1,75 @@
+import type { BaseItemDto } from '@jellyfin/sdk/lib/generated-client/models/base-item-dto';
+import { BaseItemKind } from '@jellyfin/sdk/lib/generated-client/models/base-item-kind';
+import type { ApiClient } from 'jellyfin-apiclient';
+import type { CardOptions } from 'types/cardOptions';
+import { CardShape } from 'utils/card';
+
+import { getCardImageUrl, getDefaultText } from './cardBuilder';
+
+/**
+ * Builds an html string for a basic image only card.
+ */
+export function buildCardImage(
+    apiClient: ApiClient,
+    item: BaseItemDto,
+    options: CardOptions
+): string {
+    let shape: CardShape = CardShape.Square;
+    if (item.PrimaryImageAspectRatio) {
+        if (item.PrimaryImageAspectRatio >= 3) {
+            shape = CardShape.Banner;
+        } else if (item.PrimaryImageAspectRatio >= 1.33) {
+            shape = CardShape.Backdrop;
+        } else if (item.PrimaryImageAspectRatio > 0.71) {
+            shape = CardShape.Square;
+        } else {
+            shape = CardShape.Portrait;
+        }
+    }
+
+    const image = getCardImageUrl(
+        item,
+        apiClient,
+        options,
+        shape
+    );
+
+    if (!image) return '';
+
+    const className = ` ${shape}Card`;
```

Comment:
_nitpicking_
```suggestion
    const className = `${shape}Card`;
```
The space is in the output string.

Reply:
I just removed this variable since it is only used in one place anyway